unit2/unit2.ipynb

```diff
@@ -315,7 +315,7 @@
       "outputs": [],
       "source": [
         "# Create the FrozenLake-v1 environment using 4x4 map and non-slippery version\n",
-        "env = gym.make()"
+        "env = gym.make() # TODO use the correct parameters"
```

Comment:
I didn't realize I needed to edit this code. The later code was more clear because the python syntax wasn't filled in. i.e. `action =` on a line clearly needs to be modified.